certified-connectors/PIPware KPIs/apiDefinition.swagger.json

```diff
@@ -0,0 +1,220 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "PIPware KPIs",
+    "description": "This connector allows you to save KPI Target or KPI Actual data to your PIPware instance. This is more efficient than capturing the data into PIPware manually. Commonly used data sources are Excel or SQL, but any source returning data in the expected format can be used.",
+    "contact": {
+      "name": "PIPware Support",
+      "url": "https://support.pipware.net",
+      "email": "support@pipware.net"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.pip.global/en/services/pipware"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": ""
```

Comment:
@PIPwareInfo Please update the Privacy policy property value to "Not Available" instead of an empty string as our validation would pass "Not Available" but not an empty string..

Reply:
Hi @sriyen-msft . This is now resolved